msteams-platform/m365-apps/prerequisites.md

```diff
@@ -25,15 +25,21 @@ You need a Microsoft 365 developer subscription sandbox tenant to set up your de
 
 You'll also need to enable sideloading for your tenant:
 
-1. Sign in to Microsoft 365 admin center (https://admin.microsoft.com) with your test tenant credentials and select **Teams** from the side panel to open the *Microsoft Teams admin center*.
-1. Select: Teams apps > Manage apps > **Org-wide app settings**.
-1. Under **Custom apps**, toggle on the option *Interaction with custom apps*.
+ 1. Sign in to [Teams admin center](https://admin.teams.microsoft.com/dashboard) with your test tenant credentials.
 
-    :::image type="content" source="images/teams-admin-enable-sideloading.png" alt-text="Enable sideloading for custom apps from the Teams admin center":::
+ 1. Go to **Teams apps** > **Manage apps**.
+
+ 1. At the top right, select **Org-wide app settings**.
+
+ 1. Under Custom apps, turn on the **"Interaction with custom app"** toggle and save.
+
+    :::image type="content" source="images/teams-admin-enable-sideloading.png" alt-text="The screenshot is an example that enables sideloading for custom apps from the Teams Admin Center":::
+
+ 1. Apart from Org-wide app settings, app setup policies also allow users to upload custom apps to Teams. Enable the **Upload custom apps** option in **App setup policies**. For more information, see [Upload custom apps setting](/microsoftteams/upload-custom-apps).
```

Comment:
just add the following steps

1. In the left navigation of the Microsoft Teams admin center, go to Teams apps > Setup policies, and then select the Global (Org-wide default) policy.

1. Turn on Upload custom apps, and then select Save.

Reply:
Added the steps as suggested.